js/everything.js

```diff
@@ -630,6 +641,23 @@ $(function() {
     });
     // Load the player
     ZenPlayer.init(currentVideoID);
+
+    $(document).on("keyup", function(evt) {
+        evt.preventDefault();
+        if (evt.keyCode === 32) {
+            if (ZenPlayer.isPlaying) {
+                ZenPlayer.pause();
+            }
+            else {
+                ZenPlayer.play();
+            }
+        }
+    });
+
+        // event handler for keydown , prevent default scrolling handler
+    $(document).on("keydown", function(evt) {
+        evt.preventDefault();
```

Comment:
@jeanPokou without the conditional, this is causing many issues (arrow keys, tab, all alphanumeric, etc. keys are being swallowed).

We also want to make sure we don't play/pause while the user is typing in the the search box